packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -149,8 +149,26 @@ function nextHook(): null | Hook {
 }
 
 function readContext<T>(context: ReactContext<T>): T {
+  if (context === GetPrimitiveStackCacheContext) {
+    // This is a read for filling the primitive stack cache.
+    // There's no sensible value to return.
+    return (null: any);
+  }
+  if (currentFiber === null) {
+    // Hook inspection without access to the Fiber tree.
+    return context._currentValue;
```

Comment:
This is only for `ReactDebugTools.inspectHooks` where we don't have access to the current fiber (we have tests that would break if we throw just because `currentContextDependency` is null). Which also means that `ReactDebugTools.inspectHooks` would not have support for `useFormStatus`.

Should we leave a comment here highlighting the issues with `ReactDebugTools.inspectHooks`, remove that API entirely or did I misunderstand what the desired implementation is?